docs/Makefile

```diff
@@ -18,3 +18,7 @@ help:
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
 %: Makefile
 	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+clean:
+	-rm -rf $(BUILDDIR)/*
+	-rm -rf source/reference/api/api_rst/
```

Comment:
use `SOURCEDIR` rather than `source`?

Reply:
Also shall we put the generated files to an obvious position?
we cannot know that this folder contains generated files from its name `source/reference/api/api_rst/`.

Reply:
So we could use file name "generated" to refer to automatically generated files？

Reply:
I think it is OK to use "generated"